unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -1451,15 +1451,15 @@ public static void NotifyStatusThroughChat(string message)
             SendMessage("VideoProgressEvent", progressEvent);
         }
 
-        public static void ReportAvatarRemoved(string entityId, string avatarId)
+        public static void ReportAvatarRemoved(long entityId, string avatarId)
         {
             avatarStatePayload.type = "Removed";
             avatarStatePayload.entityId = entityId;
```

Comment:
should this type change be considered in kernel?

Reply:
Yes, this type should change in kernel as well

Reply:
I propose you to create a different function with the new types so we don't break backwards compatibility

Reply:
After creating another function with new types I just realized that this field has not usage on the kernel side, I have made a PR to remove that field in kernel and removed it from this part as well